packages/xrpl/test/integration/index.ts

```diff
@@ -23,24 +26,30 @@ export * from './requests/accountTx.test'
 export * from './requests/bookOffers.test'
 export * from './requests/channelVerify.test'
 export * from './requests/depositAuthorized.test'
+export * from './requests/fee.test'
 export * from './requests/gatewayBalances.test'
 export * from './requests/ledger.test'
 export * from './requests/ledgerClosed.test'
 export * from './requests/ledgerCurrent.test'
 export * from './requests/ledgerData.test'
 export * from './requests/ledgerEntry.test'
-export * from './requests/submitMultisigned.test'
 export * from './requests/noRippleCheck.test'
 export * from './requests/pathFind.test'
 export * from './requests/ripplePathFind.test'
+export * from './requests/serverInfo.test'
 export * from './requests/submit.test'
+export * from './requests/submitMultisigned.test'
 export * from './requests/subscribe.test'
 export * from './requests/tx.test'
 export * from './requests/utility.test'
 
 export * from './fundWallet.test'
 export * from './integration.test'
+export * from './onConnect.test'
 export * from './regularKey.test'
+export * from './submitAndWait.test'
```

Comment:
Relatedly, `./wallet.test.ts` doesn't really seem to be a test - did the name change in this PR? (Is there a reason it has a `.test.ts` end?)

Not sure if it should be included here or not.

Reply:
It should be included. All test files now have .test for consistency